package.json

```diff
@@ -24,47 +23,44 @@
     "@material-ui/icons": "4.9.1",
     "@material-ui/lab": "^4.0.0-alpha.56",
     "axios": "0.19.0",
-    "json-server": "0.15.1",
-    "npm-run-all": "4.1.5",
     "prop-types": "^15.7.2",
     "react": "16.11.0",
     "react-dom": "16.11.0",
     "react-hook-form": "5.5.1",
-    "react-query": "^2.5.14",
+    "react-query": "^2.15.4",
     "react-router-dom": "5.1.2",
     "react-scripts": "^3.4.0",
-    "react-select": "3.0.8",
-    "unfurl.js": "5.0.1"
+    "react-select": "3.0.8"
   },
   "devDependencies": {
-    "@babel/core": "^7.8.4",
-    "@babel/plugin-transform-runtime": "^7.9.0",
-    "@babel/preset-env": "^7.8.4",
+    "@babel/core": "^7.11.6",
+    "@babel/plugin-transform-runtime": "^7.11.5",
+    "@babel/preset-env": "^7.11.5",
     "@babel/preset-react": "^7.8.3",
     "@sheerun/mutationobserver-shim": "^0.3.3",
     "@storybook/addon-actions": "^5.3.9",
     "@storybook/addon-links": "^5.3.9",
     "@storybook/addons": "^5.3.9",
     "@storybook/preset-create-react-app": "^1.5.2",
     "@storybook/react": "^5.3.19",
-    "@testing-library/jest-dom": "^5.1.1",
-    "@testing-library/react": "^10.0.3",
-    "@testing-library/user-event": "^12.0.11",
+    "@testing-library/jest-dom": "^5.11.4",
+    "@testing-library/react": "^10.4.9",
+    "@testing-library/user-event": "^12.1.4",
     "eslint": "^6.8.0",
     "eslint-config-prettier": "^6.10.0",
     "eslint-config-semistandard": "^15.0.0",
```

Comment:
eslint-config-semistandard@15.0.1 requires a peer of eslint-config-standard@>=14.1.0 but none is installed. You must install peer dependencies yourself.

Reply:
Peer Dependencies are typically coming from packages and their own development. It's most likely not related to us here and is safe to ignore.